server/src/main/java/io/spine/server/aggregate/Aggregate.java

```diff
@@ -203,6 +214,7 @@ protected B getBuilder() {
      */
     @Override
     protected List<? extends Message> dispatchCommand(CommandEnvelope command) {
+        idempotencyGuard.ensureIdempotence(command);
```

Comment:
This is hard to read. We have to read `idempotent` twice. Does the guard have other public methods for checking?

Can it be something like?

```java
idempotencyGuard.check(command);
```